### ISSUE TITLE
create design pattern for interaction relations

### ISSUE THREAD

**Issue by cmungall** [oborel/obo-relations#211]
These are done manually. The pattern is partially documented in the wiki, but good to formalize. Typically we have a process that corresponds to the interaction, e.g. symbiosis, and use the self pattern (should use swrl now?).

This may be a good candidate for a powerful templating framework, e.g tawny/scowl

**Comment by nlharris** [comment]
Is this still something we want done?

**Comment by cmungall** [comment]
yes, see my comments https://github.com/oborel/obo-relations/pull/522
